src/CommunityToolkit/Xamarin.CommunityToolkit/Extensions/JavaObjectExtensions.android.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace Xamarin.CommunityToolkit.Extensions
+{
+	internal static class JavaObjectExtensions
+	{
+		public static bool IsDisposed(this Java.Lang.Object obj)
+		{
+			return obj.Handle == IntPtr.Zero;
+		}
```

Comment:
```suggestion
			=> obj.Handle == IntPtr.Zero;
```

Reply:
Should we check for null here as well?

Reply:
I am no sure I took it as it is from XF team https://github.com/xamarin/Xamarin.Forms/blob/80a4dbfb1cd897f478c38d0aafa6a5d67940d2f3/Xamarin.Forms.Platform.Android/Extensions/JavaObjectExtensions.cs it was suggested by Shane link of the conversation in the issue.